problems/isOdd.js

```diff
@@ -7,8 +7,12 @@
 
 */
 
-function isOdd() {
-
+function isOdd(n) {
+  if (n % 2 == 1) {
```

Comment:
If we used implicit return in this function do we need an if-else statement? This code can be condensed.